lib/rust/lexer/definition/src/lexer.rs

```diff
@@ -90,6 +101,7 @@ impl EnsoLexer {
 
     /// Consume the currently active stream of tokens.
     pub fn consume_tokens(&mut self) -> token::Stream {
+        debug!(self.logger,"Consume Tokens: {&self.output:?}");
```

Comment:
Please make all (also not included in this commit) debug messages sentences - ending with dot.